features/steps/test_starter/{{ cookiecutter.repo_name }}/conf/base/logging.yml

```diff
@@ -34,7 +34,9 @@ handlers:
     delay: True
 
   rich:
-    class: rich.logging.RichHandler
+    class: kedro.extras.logging.RichHandler
+    rich_tracebacks: True
+    # tracebacks_show_locals: False
```

Comment:
Like `settings.py`, we could have some commented default here to help user discover possible options. I put `show_locals` because initially when we introduce `rich`, we want to enable it as `True`. It turns out to be a bit noisy for some users, and there was no good way to turn it off by the user.